src/planwise/client/effects.cljs

```diff
@@ -112,7 +111,7 @@
  :api-abort
  (fn [key]
    (some-> (get-in @rf-db/app-db key) ajax.protocols/-abort)
-   (swap! rf-db/app-db assoc-in key nil)))
+   (swap! rf-db/app-db dissoc key)))
```

Comment:
As above, and we would also need to dissoc the key when the request finishes correctly.